ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_and_vector_products.py

```diff
@@ -197,3 +197,36 @@ def test_numpy_kron(
         a=xs[0],
         b=xs[1],
     )
+
+#multi_dot
+@handle_frontend_test(
+    fn_tree="numpy.linalg.multi_dot",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=5,
+        min_num_dims=2,
+        max_num_dims=2,
+        min_dim_size=2,
+        max_dim_size=2,
+        min_value=-100,
+        max_value=100,
+        shared_dtype=True,
+    )
```

Comment:
The function supports broadcasting, so more exhaustive generation should be used. I would recommend you just import the following `_generate_multi_dot_dtype_and_arrays` from here: https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py#L530

Use that to generate dtype_and_x